docs/self-managed/zeebe-deployment/configuration/gateway.md

```diff
@@ -0,0 +1,277 @@
+---
+id: gateway-config
+title: "Gateway configuration"
+sidebar_label: "Gateway configuration"
+description: "Analyze how to configure the Zeebe gateway, including byte sizes, time units, paths, and sample YAML snippets."
+---
+
+A complete gateway configuration template is available in the [Zeebe repository](https://github.com/camunda/zeebe/blob/main/dist/src/main/config/gateway.yaml.template).
+
+## Conventions
+
+Take the following conventions into consideration when working with the gateway configuration.
+
+### Byte sizes
+
+For buffers and others must be specified as strings and follow the following format: "10U" where U (unit) must be replaced with KB = Kilobytes, MB = Megabytes or GB = Gigabytes. If unit is omitted, the default unit is bytes.
```

Comment:
"For buffers and others must be specified as strings..." -- what do we mean by "others" here?

Reply:
from my understanding that is a general wildcard for anything that requires some data size, e.g. also size limits such as [`maxMessageSize`](https://github.com/camunda/zeebe/blame/main/dist/src/main/config/gateway.yaml.template#L53).
Buffers is essentially just an example, given this do you have a suggestion on how to best rephrase?

Reply:
Maybe something like "For buffers and other data values that include sizing..."

Reply:
@christinaausley do you have a suggestion for phrasing here? My suggestion still feels a bit awkward.

Reply:
Perhaps "Buffers and data values referencing sizing must be specified as strings..."